### PR TITLE
hotfix/ci

### PR DESCRIPTION
**Pull request by aabounegm**
Fixes Pylint issues by adding docstrings and applying various other fixes.
Also downgrades Python in the workflow to 3.8

### CODE REVIEW

File: vpt/cli/sources.py [hunk 1]
```diff
@@ -48,8 +48,7 @@ def list_video_inputs():
         cap = cv2.VideoCapture(index)
         if not cap.read()[0]:
             break
-        else:
-            arr.append(index)
+        arr.append(index)
```
Comment (illright) [inline vpt/cli/sources.py:51]: Why this?

File: requirements.txt [hunk 2]
```diff
@@ -4,7 +4,7 @@ sounddevice~=0.4.1
 keyboard~=0.13.5
 mouse~=0.7.1
 SoundFile~=0.10.3.post1
-numpy~=1.19.5
+numpy~=1.17.3
 ffmpeg~=1.4
 tensorflow~=2.3.2
-#tensorflow-gpu~=2.3.2
+matplotlib~=3.3.4
```
Comment (illright) [inline requirements.txt:10]: Can we also switch to Pipenv with this PR? please please

File: vpt/processors/gaze_detector/__init__.py [hunk 3]
```diff
@@ -92,7 +92,7 @@ def get_square_box(box):
 
         if diff == 0:  # Already a square.
             return box
-        elif diff > 0:  # Height > width, a slim box.
+        if diff > 0:   # Height > width, a slim box.
```
Comment (illright) [inline vpt/processors/gaze_detector/__init__.py:95]: Why this?

File: vpt/cli/sources.py (hunk 1 above)
Comment (aabounegm) [reply vpt/cli/sources.py:51]: Because there is a `break` in the `if` branch. Pylint recommended it

File: requirements.txt (hunk 2 above)
Comment (aabounegm) [reply requirements.txt:10]: Would we all **have** to use Pipenv, or will there still be a `requirements.txt`?
Anyways, I don't mind, but you will do it. And I recommend for that to be in a separate PR

File: vpt/processors/gaze_detector/__init__.py (hunk 3 above)
Comment (aabounegm) [reply vpt/processors/gaze_detector/__init__.py:95]: Just like with the `break`, there is a `return` in the `if` branch here, meaning the `else` is useless

File: vpt/cli/sources.py (hunk 1 above)
Comment (illright) [reply vpt/cli/sources.py:51]: Oh, right

File: requirements.txt (hunk 2 above)
Comment (illright) [reply requirements.txt:10]: Yeah, Pipenv is a replacement for `requirements.txt`. Alright then, I'll make a PR, see what people think

File: vpt/processors/gaze_detector/__init__.py (hunk 3 above)
Comment (illright) [reply vpt/processors/gaze_detector/__init__.py:95]: Got it

**Review by illright — APPROVED** [review]

**Review by alkaitagi — APPROVED** [review]

**Review by IamMaxim — APPROVED** [review]

**Review by kuredoro — APPROVED** [review]

**Pull request merged**